### PR TITLE
spiderAgent: install cni,ovs and rdma in init-container

### PR DESCRIPTION
**Pull request by cyclinder**
---
name: Pull request
about: Tell us about your contribution
labels: ["pr/release/none-required"]
---

---
## Thanks for contributing !

Before submitting a pull request, make sure you read about our Contribution notice here: <https://spidernet-io.github.io/spiderpool/latest/develop/contributing/>

**What this PR does / why we need it**:

spiderAgent: install cni,ovs and rdma in init-container

**Which issue(s) this PR fixes**:

**Special notes for your reviewer**:

**make sure your commit is signed off**

### CODE REVIEW

**Comment by cyclinder** [comment]
waitting for  https://github.com/spidernet-io/spiderpool/pull/2457

File: charts/spiderpool/values.yaml [hunk 1]
```diff
@@ -282,6 +258,37 @@ multus:
...
+    repository:  spidernet-io/spiderpool/plugins
+
+    ## @param plugins.image.pullPolicy the image pullPolicy of plugins
+    pullPolicy: IfNotPresent
+
+    ## @param plugins.image.digest the image digest of plugins
+    digest: ""
+
+    ## @param plugins.image.tag the image tag of plugins
+    tag: latest
```
Comment (weizhoublue) [inline charts/spiderpool/values.yaml:287]: 有个具体的版本号

**Comment by weizhoublue** [comment]
还有一些的 doc，不需要 从官方 下载 binary， 在安装 spiderpool 时 加个 note， 可用什么选项 一起 安装

**Comment by cyclinder** [comment]
```
 > Enter [BeforeEach] Automatic creation, extension and deletion of ippool by different controllers - /home/runner/work/spiderpool/spiderpool/test/e2e/subnet/subnet_test.go:253 @ 10/28/23 09:14:12.887
  [FAILED] Unexpected error:
      <*errors.StatusError | 0xc000362be0>: 
      spidersubnets.spiderpool.spidernet.io "v4-ss-12c41a1db8ae2c3" already exists
      {
          ErrStatus: {
              TypeMeta: {Kind: "", APIVersion: ""},
              ListMeta: {
                  SelfLink: "",
                  ResourceVersion: "",
                  Continue: "",
                  RemainingItemCount: nil,
              },
              Status: "Failure",
              Message: "spidersubnets.spiderpool.spidernet.io \"v4-ss-12c41a1db8ae2c3\" already exists",
              Reason: "AlreadyExists",
              Details: {
                  Name: "v4-ss-12c41a1db8ae2c3",
                  Group: "spiderpool.spidernet.io",
                  Kind: "spidersubnets",
                  UID: "",
                  Causes: nil,
                  RetryAfterSeconds: 0,
              },
              Code: 409,
          },
      }
  occurred
  In [BeforeEach] at: /home/runner/work/spiderpool/spiderpool/test/e2e/subnet/subnet_test.go:260 @ 10/28/23 09:14:13.065
  < Exit [BeforeEach] Automatic creation, extension and deletion of ippool by different controllers - 
```

**Pull request merged**